ext/mysqlnd/mysqlnd_auth.c

```diff
@@ -26,6 +26,9 @@
 #include "mysqlnd_priv.h"
 #include "mysqlnd_charset.h"
 #include "mysqlnd_debug.h"
+#include <krb5/krb5.h>
+#include <sasl/sasl.h>
+#include <profile.h>
```

Comment:
If new dependency libraries are introduced, their availability should be checked in ext/mysqlnd/config9.m4 (see e.g. ext/ldap/config.m4 for libsasl), and in this case the dependencies should be optional, I think, i.e. `#ifdef` the respective code.